internal/vendor/wiz.go

```diff
@@ -423,6 +450,20 @@ type KubernetesCluster struct {
 	IsConnectedUsingBroker bool         `json:"isConnectedUsingBroker"`
 }
 
+// ProjectKubernetesClusterLink struct
+type ProjectKubernetesClusterLink struct {
+	ID                string                   `json:"id"`
+	KubernetesCluster ProjectKubernetesCluster `json:"kubernetesCluster"`
+	Environment       string                   `json:"environment"`
+	Namespaces        []string                 `json:"namespaces"`
```

Comment:
This field is optional in the GraphQL schema.

```suggestion
	Namespaces        []string                 `json:"namespaces.omitempty"`
```

If there is a need to deviate from the GraphQL schema, please make a comment in the strct comments.